lib/action_client/base.rb

```diff
@@ -74,7 +95,14 @@ def build_request(method:, path: nil, url: nil, headers: {}, **options)
         request.headers[key] = value
       end
 
-      SubmittableRequest.new(@middleware, request.env)
+      app = middleware.build(ActionClient::Applications::Net::HttpClient.new)
+
+      callbacks = after_actions + [block]
```

Comment:
Probably worth documenting the order of these in the `README`

Reply:
Explained in 2a0c43a.